### ISSUE TITLE
Consider giving more visibility to the `build()` method in documentation

### ISSUE THREAD

**Issue by laynor** [shepmaster/snafu#267]
Took me a while to discover it, might be nice including it in one of the basic examples

**Comment by shepmaster** [comment]
Yeah, `build` is relatively new. It is indirectly demonstrated in https://docs.rs/snafu/0.6.9/snafu/guide/examples/basic/index.html

Out of curiosity, how are you using `build`? It's needed in some cases, but not in many others.
